app/javascript/components/visualization/StudyViolinPlot.js

```diff
@@ -67,12 +72,19 @@ export default function StudyViolinPlot({ study, gene }) {
             className="gene-load-spinner"
           />
         }
+        { isCollapsedView && studyGeneNames.length &&
+          <div className="text-center">
+            <span>{collapseBy} expression of {studyGeneNames.join(', ')}</span>
```

Comment:
Let's use [`capitalize`](https://esamattis.github.io/underscore.string/#capitalize-string-lowercaserest-false-gt-string) and [`toSentenceSerial`](https://esamattis.github.io/underscore.string/#tosentenceserial-array-delimiter-lastdelimiter-gt-string) from the `underscore.string` package:

```suggestion
            <span>{capitalize(collapseBy)} expression of {toSentenceSerial(studyGeneNames)}</span>
```

Reply:
It'd be nice polish, but I don't feel strongly about it.